src/proto/h1/role.rs

```diff
@@ -170,7 +171,11 @@ where
             extend(dst, head.subject.canonical_reason().unwrap_or("<none>").as_bytes());
             extend(dst, b"\r\n");
         }
-        write_headers(&head.headers, dst);
+        if title_case_headers {
+            write_headers_title_case(&head.headers, dst);
+        } else {
+            write_headers(&head.headers, dst);
+        }
```

Comment:
Since there's no way to set this for the server yet (and I'm tempted to leave it out until someone asks for it), the bool check is kind of a waste. Whatcha think of removing it from the `ServerTransaction` version here?

Reply:
Since `ServerTransaction` implements `Http1Transaction`, I cannot just remove it from the method signature as that would deviate from the trait definition, but I can prefix it with an underscore to signify that the boolean is unused.